### PR TITLE
lvol: Change ``pvs`` argument type to list (of str)

### LINKED ISSUE #7504: Inconsistent pvs interface in lvol and lvg
### Summary

`pvs` field of `lvg` is a list of strings. 
`pvs` field of `lvol` is a comma separated string.

### Issue Type

Bug Report

### Component Name

lvg
lvol

### Ansible Version

```console (paste below)
$ ansible --version
ansible [core 2.14.11]
  config file = /home/vscode/.ansible.cfg
  configured module search path = ['/workspaces/linux-2023-Q4/lib/ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/vscode/.local/lib/python3.11/site-packages/ansible
  ansible collection location = /workspaces/linux-2023-Q4/lib/ansible/collections:/usr/share/ansible/collections
  executable location = /home/vscode/.local/bin/ansible
  python version = 3.11.2 (main, Sep 12 2023, 00:00:00) [GCC 11.3.1 20221121 (Red Hat 11.3.1-4)] (/usr/bin/python3.11)
  jinja version = 3.1.2
  libyaml = True
```

### Community.general Version

```console (paste below)
$ ansible-galaxy collection list community.general
# /home/vscode/.local/lib/python3.11/site-packages/ansible_collections
Collection        Version
----------------- -------
community.general 6.6.1  
```

### Configuration

_No response_

### OS / Environment

_No response_

### Steps to Reproduce

1. https://docs.ansible.com/ansible/latest/collections/community/general/lvg_module.html#parameter-pvs
3. https://docs.ansible.com/ansible/latest/collections/community/general/lvol_module.html#parameter-pvs

### Expected Results

both are a list of string, obviously

### Actual Results

confusion

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by lszomor**
##### SUMMARY
I've modified the ``pvs`` argument's type to list (of str) of the ``lvol`` module to provide a consistent interface with the ``lvg`` module.
I consider this as a minor change as the current playbooks are not affected by this change due to the nature of the list type handling in the core ``AnsibleModule``.

Fixes #7504

##### ISSUE TYPE
- Bugfix Pull Request
- Refactoring Pull Request

##### COMPONENT NAME
``lvol``

### CODE REVIEW

File: changelogs/fragments/7676-lvol-pvs-as-list.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - lvol - Change ``pvs`` argument type to list (of str). (https://github.com/ansible-collections/community.general/pull/7676, https://github.com/ansible-collections/community.general/issues/7504).
```
Comment (felixfontein) [inline changelogs/fragments/7676-lvol-pvs-as-list.yml:2]:
```suggestion
  - lvol - change ``pvs`` argument type to lists of stinrgs (https://github.com/ansible-collections/community.general/pull/7676, https://github.com/ansible-collections/community.general/issues/7504).
```

File: plugins/modules/lvol.py [hunk 2]
```diff
@@ -75,9 +75,10 @@
     description:
     - The name of a snapshot volume to be configured. When creating a snapshot volume, the O(lv) parameter specifies the origin volume.
   pvs:
-    type: str
+    type: list
+    elements: str
     description:
-    - Comma separated list of physical volumes (e.g. /dev/sda,/dev/sdb).
+    - List of physical volumes (e.g. /dev/sda,/dev/sdb).
```
Comment (felixfontein) [inline plugins/modules/lvol.py:81]:
```suggestion
    - List of physical volumes (for example V(/dev/sda, /dev/sdb)).
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Since the module already allowed comma-separated lists here, this is indeed no problem.

Can you maybe also change the example in line 114 to use a proper YAML list?

**Comment by lszomor** [comment]
Thank you for reviewing! 
I've applied the suggested changes.

**Review by felixfontein — APPROVED** [review]
Looks good to me! If nobody objects, I'll merge this tomorrow.

**Pull request merged**